app/controllers/application_controller.rb

```diff
@@ -1763,14 +1763,19 @@ def get_view(db, options = {})
     @targets_hash             = attrs[:targets_hash] if attrs[:targets_hash]
 
     # Set up the grid variables for list view, with exception models below
-    if !%w(Job MiqProvision MiqReportResult MiqTask).include?(view.db) &&
-       !view.db.ends_with?("Build") && !@force_no_grid_xml && (@gtl_type == "list" || @force_grid_xml)
+    if grid_hash_conditions(view)
```

Comment:
@martinpovolny Thanks for your quick replay. I added this function here, is that what you meant? If so I would add a spec for it in spec/controllers/application_controller_spec.rb